deepfence_frontend/apps/dashboard/src/features/settings/pages/UserManagement.tsx

```diff
@@ -233,18 +233,17 @@ export const action = async ({
     if (ApiError.isApiError(r)) {
       return r.value();
     }
+    let data = {};
     if (body.intent == ModelInviteUserRequestActionEnum.GetInviteLink) {
       r.invite_url && navigator.clipboard.writeText(r.invite_url);
-      toast.success('User invite URL copied');
-      return {
-        ...r,
-        success: true,
-      };
+      data = { ...data, ...r, success: true };
+      toast.success('User invite URL copied !');
     } else if (body.intent === ModelInviteUserRequestActionEnum.SendInviteEmail) {
-      toast.success('User invite sent successfully');
+      data = { successMessage: 'User invite sent successfully' };
```

Comment:
same

Reply:
removed data variable and returned early